src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -60,7 +64,14 @@ protected function buildEntityQuery($match = NULL, $match_operator = 'CONTAINS')
     // the default selection handler of the entity, which the field reference
     // to, and add another logic to the query object i.e. check if the entities
     // bundle defined as group.
-    $query = $this->getSelectionHandler()->buildEntityQuery();
+    $selection_handler = $this->getSelectionHandler();
+
+    if (empty($selection_handler)) {
+      return $query = parent::buildEntityQuery($match, $match_operator);
```

Comment:
not sure we want to avoid it - can you explain why?

The way I see it - if the `$selection_handler` is missing it means something in the configuration is missing, no?